### PR TITLE
DOCS-2987: Add PrestaShop FAQ article about translating payment methods

### PR DESCRIPTION
**Pull request by Joris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

**Review by Daniel-MultiSafepay — APPROVED** [review]
@Sarah-MultiSafepay 

Would you mind double check this PR. 

Specially searching any typo error (my english is far away to be perfect) and check if the content is understandable.

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title : "Translating the module"
+meta_title: "PrestaShop 1.7 - Translating the module - MultiSafepay Docs"
+read_more: "."
+url: '/prestashop-1-7/translating-the-module/'
+---
+The PrestaShop 1.7 plugin offers the possibility to texts that belongs to the module.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:7]:
```suggestion
To translate elements of the MultiSafepay module in the PrestaShop 1.7 plugin, follow these steps:
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title : "Translating the module"
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:2]:
```suggestion
title : "Translating the MultiSafepay module"
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+meta_title: "PrestaShop 1.7 - Translating the module - MultiSafepay Docs"
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:3]:
```suggestion
meta_title: "PrestaShop 1.7 - Translating the MultiSafepay module - MultiSafepay Docs"
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+read_more: "."
+url: '/prestashop-1-7/translating-the-module/'
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:5]:
```suggestion
url: '/prestashop-1-7/translating-multisafepay-module/'
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+---
+The PrestaShop 1.7 plugin offers the possibility to texts that belongs to the module.
+
+To translate the module:
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:9]:
```suggestion
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Sign in to your [backend](/glossaries/multisafepay-glossary/#backend).
+2. Go to **Improve** > **International** > **Translations**.
+3. Under the **Modify translations** tab select **Installed module translations** for the **Type of translation** field.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:13]:
```suggestion
3. Under the **Modify translations** tab, in the **Type of translation** field, select **Installed module translations**.
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+4. Select the **MultiSafepay** module.
+5. Select the **Language**.
+6. Press on **Modify**.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:16]:
```suggestion
6. Click **Modify**.
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+7. On this page you can make the required translations
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:17]:
```suggestion
7. Enter the required translations.
```

Comment (FrancescaNCook) [reply content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:17]: @Daniel-MultiSafepay Can you send a screenshot of the page? Do you need to click **Save** or something after entering the translations?

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+
+In the specific case of the translations of the names of the payment methods:
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:20]:
```suggestion
To translate the names of payment methods:
```

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Go to **Improve** > **MultiSafepay** > **Payment methods**.
+2. Select the Payment Method you would like to translate.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:23]:
```suggestion
2. Select the payment method you want to translate.
```

**Review by FrancescaNCook — COMMENTED** [review]
@Daniel-MultiSafepay Sarah is away today so I thought I'd review to keep things moving! A few linguistic changes and one question for you!

**Comment by Daniel-MultiSafepay** [comment]
@FrancescaNCook thanks for check this. Here it is an screenshot of the translation interface. According with this I am going to add a new line in the instructions. 

screencapture-prestashop-dev-daniel-multisafepay-io-admin-dev-index-php-2022-01-28-15_58_32

**Review by Noelknae445 — APPROVED** [review]

File: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+
+1. Sign in to your [backend](/glossaries/multisafepay-glossary/#backend).
+2. Go to **Improve** > **International** > **Translations**.
+3. Under the **Modify translations** tab, in the **Type of translation** field, select **Installed module translations**.
+4. Select the **MultiSafepay** module.
+5. Select the **Language**.
+6. Click **Modify**.
+7. Enter the required translations.
+8. Press **Save** button.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md:17]:
```suggestion
8. Click **Save**. 
```

**Review by FrancescaNCook — APPROVED** [review]
@Daniel-MultiSafepay Nice one!

**Pull request merged**